package.json

```diff
@@ -4,7 +4,7 @@
   "description": "prototype for SMART Health Links",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --silent --runInBand",
```

Comment:
It might be nice to mention how to run the tests in the README, and also that you can't run the tests while simultaneously running the server.

I don't think we need a comprehensive README yet, but just quick bullets of what `npm start` and `npm test` do.

Reply:
Looks great!